src/labels/CustomLabels.labels

```diff
@@ -2699,6 +2723,22 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <value>Consider deleting these unused One-to-One Accounts to save space.</value>
     </labels>
     <labels>
+        <fullName>healthSolutionAccountDefaultRTInvalid</fullName>
+        <categories>Health Check</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>healthSolutionAccountDefaultRTInvalid</shortDescription>
+        <value>Go to &lt;strong&gt;People | Account Model&lt;/strong&gt; and select a valid Record Type.</value>
+    </labels>
+    <labels>
+        <fullName>healthSolutionAccountDefaultRTIssue</fullName>
+        <categories>Health Check</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>healthSolutionAccountDefaultRTIssue</shortDescription>
+        <value>The Record Type selected for the current Account Model should not be used as the default Account Record Type for any Profile in your org. Using the same Record Type could cause various data issues.</value>
```

Comment:
@smarovich Question for you. Do you think the the phrase `in your org` in the above label will be clear to admins who see this message, or should that be spelled out as something like `in this organization` or `in this Salesforce Organization`?

Reply:
Great point, @force2b . I use the term "org" too freely sometimes... "in this Salesforce organization" is a great alternative.